pkgs/tools/networking/picosnitch/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, fetchurl
+, python3
+, bcc
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "picosnitch";
+  version = "0.12.0";
+
+  src = fetchurl {
+    url = "https://github.com/elesiuta/picosnitch/releases/download/v${version}/picosnitch.tar.gz";
+    sha256 = "92bfa7c75d94a07669d54a62f0e4cc190c3989c7990bc286fdab9f7ae347f336";
+  };
```

Comment:
Can we use fetchFromGitHub instead? Do we need to generate some files?

Reply:
That was just for fetching the service file with the package, I changed it back to fetchPypi since I added a nixos module and it's easier for me to get the sha256 for fetchPypi than fetchFromGitHub.

Reply:
Just set the hash to an empty string and build the package. Then the error containers the correct hash for you.